src/sentry/relay/config/__init__.py

```diff
@@ -489,300 +845,12 @@ def _get_project_config(
-                        },
-                        {
-                            "measurement": "ttfb",
-                            "weight": 0.10,
-                            "p10": 200.0,
-                            "p50": 400.0,
-                            "optional": False,
-                        },
-                    ],
-                    "condition": {
-                        "op": "eq",
-                        "name": "event.contexts.browser.name",
-                        "value": "Opera",
-                    },
-                },
-                {
-                    "name": "Chrome INP",
-                    "scoreComponents": [
-                        {
-                            "measurement": "inp",
-                            "weight": 1.0,
-                            "p10": 200.0,
-                            "p50": 500.0,
-                            "optional": False,
-                        },
-                    ],
-                    "condition": {
-                        "op": "or",
-                        "inner": [
-                            {
-                                "op": "eq",
-                                "name": "event.contexts.browser.name",
-                                "value": "Chrome",
-                            },
-                            {
-                                "op": "eq",
-                                "name": "event.contexts.browser.name",
-                                "value": "Google Chrome",
-                            },
-                        ],
-                    },
-                },
-                {
-                    "name": "Edge INP",
-                    "scoreComponents": [
-                        {
-                            "measurement": "inp",
-                            "weight": 1.0,
-                            "p10": 200.0,
-                            "p50": 500.0,
-                            "optional": False,
-                        },
-                    ],
-                    "condition": {
-                        "op": "eq",
-                        "name": "event.contexts.browser.name",
-                        "value": "Edge",
-                    },
-                },
-                {
-                    "name": "Opera INP",
-                    "scoreComponents": [
-                        {
-                            "measurement": "inp",
-                            "weight": 1.0,
-                            "p10": 200.0,
-                            "p50": 500.0,
-                            "optional": False,
-                        },
-                    ],
-                    "condition": {
-                        "op": "eq",
-                        "name": "event.contexts.browser.name",
-                        "value": "Opera",
-                    },
-                },
-            ]
-        }
+    config["performanceScore"] = {
+        "profiles": [
```

Comment:
Previously, orgs without the feature flag enabled would not have `config.performanceScore.profiles` in the config. Could this still be the case?

Reply:
I've added a condition to only apply these properties to the config if we have any profiles to add. Essentially I've just written helpers that use the feature flags internally to decide which/if there are profiles